### DISCUSSION TITLE
A problem with the geo function

### DISCUSSION THREAD

**Discussion by LemonLiTree** [ClickHouse/ClickHouse#53383] [Q&A]
I have a table that stores location information, location Point;
Each row of data stores a point. 
I will have a polygon with WKT and I want to calculate the function of contain(polygon, point), returns true if the point is contained by the polygon.
Which function should I use

**Comment by UnamedRus** [comment]
You can look into pointInPolygon function + https://clickhouse.com/docs/en/sql-reference/functions/geo/polygons#readwktmultipolygon
